modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -46,29 +46,34 @@ If this step is not complete, finalizers make it difficult to fully uninstall th
 +
 [source,terminal]
 ----
-$ oc delete customresourcedefinitions.apiextensions.k8s.io workspaceroutings.controller.devfile.io
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaceroutings.controller.devfile.io
 ----
 +
 [source,terminal]
 ----
-$ oc delete customresourcedefinitions.apiextensions.k8s.io components.controller.devfile.io
+$ oc delete customresourcedefinitions.apiextensions.k8s.io devworkspaces.workspace.devfile.io
```

Comment:
It may hurt CRW. Probably we should add selector here. Not sure if you propagate devworkspace labels to related CRs, like devworkspace routing, but they should be removed automatically with devworkspace CRs by K8s since we set ownerRefs.